### DISCUSSION TITLE
Export qcloud project manually

### DISCUSSION THREAD

**Discussion by mrgrimshaw** [opengisch/QField#3844] [Q&A]
While the Qcloud is down is there a way to export collected data manually via a cable? I have 3 days of data to export and i'm worried it is going to be lost. 

Thanks

**Comment by yaoyuandejuli** [comment]
Open the project you want to fetch go to setting -> open project folder and then you can export each item . 

good luck

**Reply by mrgrimshaw** [reply]
This works great on android but it not an option on ios. Any ideas?

**Reply by mbernasocchi** [reply]
hey @mrgrimshaw since yesterday evening QFieldCloud is up and running again. A fix has been deployed; all systems are operational again and we'are monitoring the situation.

Indeed in iOS there is no export functionality yet. It's something we're assessing currently.

**Comment by 9ls1** [comment]
@mbernasocchi Any progress in this: exporting local project from iOS? This is a very nice "backup"-solution on Android if, for some reason, mismatch between local project and cloud project. See also [https://github.com/opengisch/QField/discussions/4358](https://github.com/opengisch/QField/discussions/4358) and [https://github.com/opengisch/QField/discussions/4371](https://github.com/opengisch/QField/discussions/4371) if there is way of getting the local data from an iPhone or iPad.
